src/pages/tag.js

```diff
@@ -74,6 +74,20 @@ export default class TagGridPage extends Component {
     this.props.uiStore.setSortBy(sortBy);
   };
 
+  renderHeaderMetadata = (metafields) => {
+    const metadata = [];
+    metafields.forEach((field) => {
+      if (field.valueType && field.valueType === "header") {
+        const metadatum = {};
+        const { scope } = field;
+        metadatum[scope] = field.key;
+        metadatum.content = field.value;
```

Comment:
Since `scope` is de-structured, then perhaps the other properties like `key`, `value`, and maybe even `valueType` / `namespace` should also be de-structured.